docs/modding/suitmodding/addinglocals.mdx

```diff
@@ -0,0 +1,666 @@
+2. **Repeat [the steps from start](#1-identify-your-locr)** (or continue here)
+2. **Add a New** <Tooltip text={<strong>LOCR Array</strong>} tooltip="an array in the manifest resembling the game's LOCR file syntax" />: Use the same structure, making sure to follow the <Tooltip text={<strong>syntax rules</strong>} tooltip="A set of rules that define how code is written and organized." />.
+3. <Tooltip text={<strong>Repeat the Conversion</strong>} tooltip="Using the Localisation Hash calculator" />: Use <Tooltip text={<strong>Text Tools</strong>} tooltip="A tab in GlacierKit that provides tools for converting text and hashes for game modifications." /> again to convert any new <Tooltip text={<strong>strings</strong>} tooltip= "A key that links to text in the LOCR File" /> into <Tooltip text={<strong>decimals</strong>} tooltip="A string converted into a numeral form" />.
+4. **Update the** <Tooltip text={<strong>manifest</strong>} tooltip="A file in your mod folder that holds important information and can execute code" />: Add the new language entries, keeping everything consistent.
+
+Example of every aspect from this tutorial applied:
+```json
+"localisationOverrides": {
+    "yourLocrHASH": {
+        "english": {
+            "yourconverteddecimal": "New Suit Name",
+            "yourotherconverteddecimal": "New Suit Description/or Other New Suit name"
+        }
+    },
+    "otherLocrHASH": {
+        "english": {
+            "somenewconverteddecimal": "More New Suit Name",
+            "someotherconverteddecimal": "More New Suit Description/or Other More New Suit name"
+        },
+        "spanish": {
+            "somenewconverteddecimal": "Más nombre de traje nuevo",
+            "someotherconverteddecimal": "Más descripción del traje nuevo/u otro nombre del traje más nuevo"
+        }
+    }
+}
+```
+
+*Remember to keep your* <Tooltip text={<strong>manifest</strong>} tooltip="A file in your mod folder that holds important information and can execute code" /> *clean and organized to avoid errors.*
+
+---
+
+## Advanced Search Guide
+[[Return to tutorial]](#3-confirm-your-locr)
+
+*When you need to find text for something other than suits, use the* <Tooltip text={<strong>Advanced Search Tab</strong>} tooltip="A tab in GlacierKit for performing extended searches that go beyond the 'game contents' tab search abilities. It has a magnifying glass icon and, is like all other tabs, on the left side." />. 
+
+Start with:
+
+1. Clicking on the <Tooltip text={<strong>Advanced Search Tab</strong>} tooltip="A tab in GlacierKit for performing extended searches. Located on the left." />
+
+2. **Set Your Search Parameters**:
+   - Check only the `chunk0` box.
```

Comment:
This is not generally good advice, while suit localisation is in chunk0 you don't want people to end up thinking of chunk0 as a magical box they just need to tick, because then they'll try to search for things in Mendoza and fail